contrib/hbwin/hbwin.hbp

```diff
@@ -29,7 +29,7 @@ mapi.c
 oemansi.c
 olecore.c
 oleuuid.c{watcom}
-hbolesrv.c
+{HBWIN_HAS_OLESRV}hbolesrv.c
```

Comment:
If I set _HB_DYNSUFF and recompile (latest trunk) a simple example I get

ld.exe: cannot find -lhbwin-32

Interesting enough, this flag has an impact on a small support library called hbwin32 ( which is local and adds some missing Win32 functionality ) 

ld.exe: cannot find -lhbwin32-32